spec/queries/find_members_with_relationship_spec.rb

```diff
@@ -7,16 +7,16 @@
 
   describe "#find_members_with_relationship" do
     it "returns members with `loaded_relationship` loaded" do
```

Comment:
How are these changes related to the `prepopulate` change?

Reply:
Oh! This one was interesting - it wasn't pre-populating before. It's actually impossible to save a change_set with two genres because it's marked as a single field in the ChangeSet. Now that it happens on initialize, I had to use a field which was marked as multiple to get two things to show up.